icekit/workflow/models.py

```diff
@@ -69,6 +69,14 @@ def __str__(self):
 
 class WorkflowStateMixin(models.Model):
     workflow_states = GenericRelation(WorkflowState)
+    brief = models.TextField(
+        blank=True,
+        help_text="A document brief describing the purpose of this item"
```

Comment:
Only quibble is the use of the term 'item'. I'd use 'content'. Only saying so because of how many migrations it affects.